spec/requests/api/v1/passwords/create_spec.rb

```diff
@@ -0,0 +1,37 @@
+require 'rails_helper'
+
+describe 'POST api/v1/users/passwords', type: :request do
+  let!(:user) { create(:user) }
+
+  context 'with valid email params' do
+    let(:params) { { email: user.email } }
+
+    it 'returns a successful response' do
+      post user_password_path, params: params, as: :json
+      expect(response).to have_http_status(:success)
+    end
+
+    it 'returns email sent notification' do
+      post user_password_path, params: params, as: :json
+      expect(json[:message]).to match(user.email)
```

Comment:
Here I think that you are not expecting what the example is describing. Maybe you should expect the full notification or change the description.